### PR TITLE
Bump version (v3.1.1)

### PR DESCRIPTION
**Pull request by Farfurix**
### Changes
1. Remove `Babel`, `eval` hacks and the `build` task
2. Move code base to `\lib`
3. Use default ESLint parser

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "parser": "@babel/eslint-parser",
+  "parser": "esprima",
```
Comment (miherlosev) [inline .eslintrc:2]: Can we use the default eslint parser (`Espree`)?

File: lib/index.js [hunk 2]
```diff
@@ -1,6 +1,11 @@
-import { Selector } from 'testcafe';
+'use strict';
 
-export default Selector(complexSelector => {
+exports.__esModule = true;
```
Comment (miherlosev) [inline lib/index.js:3]: Can we use the regular import style - `module.exports = Selector(...)`

File: lib/index.js [hunk 3]
```diff
@@ -1,6 +1,11 @@
...
+exports['default'] = void 0;
+
+var Selector = require('testcafe').Selector;
```
Comment (miherlosev) [inline lib/index.js:6]: `const { Selector } = require('testcafe')`

Ensure the [no-vars](https://eslint.org/docs/rules/no-var) rule.

**Review by miherlosev — CHANGES_REQUESTED** [review]

File: .eslintrc (hunk 1 above)
Comment (Farfurix) [reply .eslintrc:2]: I mixed up their names, I'll delete this line to use the default `Espree` parser.

File: lib/index.js [hunk 4]
```diff
@@ -1,9 +1,9 @@
-import { Selector } from 'testcafe';
+const Selector = require('testcafe').Selector;
```
Comment (miherlosev) [inline lib/index.js:1]: `const { Selector } = require('testcafe')`

**Review by miherlosev — CHANGES_REQUESTED** [review]

**Review by miherlosev — APPROVED** [review]

**Pull request merged**